src/main/java/Duke.java

```diff
@@ -1,10 +1,139 @@
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.util.ArrayList;
+
+
 public class Duke {
-    public static void main(String[] args) {
+
+    public static void printIntro(){
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void main(String[] args) throws UnknownCommandException,EmptyInputException,java.io.IOException{
+        printIntro();
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> list = new ArrayList<>();
+        int count=0;
+        //Task[] list = new Task[100];
```

Comment:
Avoid leaving commented code in your codebase. If you require this fragment in the future, you can retrieve it using version control. The danger of leaving it in is that it may be left there permanently